dev-db/sqlitebrowser/sqlitebrowser-3.10.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils eutils
+
+DESCRIPTION="SQLite Database Browser"
+HOMEPAGE="http://sqlitebrowser.org"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3 MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="
+	dev-cpp/antlr-cpp:2
+	dev-db/sqlite:3
+	>=dev-libs/qcustomplot-2.0.0[qt5(+)]
+	dev-qt/qtgui:5
```

Comment:
Also needs qtcore:5 for sure.

Reply:
It does, but for some reason that seems to be missing from the 3.9.1-r1 I used as a base.
Adding...